PCbuild/lib.pyproj

```diff
@@ -1263,7 +1262,6 @@
     <Compile Include="test\test_signal.py" />
     <Compile Include="test\test_site.py" />
     <Compile Include="test\test_slice.py" />
-    <Compile Include="test\test_smtpd.py" />
```

Comment:
Whilst it's still needed and used (private or not), should it still be tested? And remove the test when the (private) module is also removed?

(This PR has a conflict.)

Reply:
> should it still be tested

I didn't initially notice that private modules have tests too so thank you for pointing this out. As a solution, I moved the module into `Lib/test/smtpd.py` for further processing.

I have a plan to gradually port the 885-line `_smtpd` to asyncio in reviewable chunks (stripping it by the way), then finally move the leftovers into test_smtplib. Having tests removed makes this process possible.

> (This PR has a conflict.)

Fixed.